react-frontend/src/components/TextAreaInput/TextAreaInput.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './TextAreaInput.css'
+
+const TextAreaInput = ({ label, value, onChange }) => {
+  return (
+    <div className="textarea-input">
+      <label>{label}</label>
```

Comment:
Please use `for` (see comments above)

Reply:
Refactored [0c2832f](https://github.com/YaroslavKSE/AuctionWeb/pull/2/commits/0c2832f95bc0ca4eb2cb355a368e619b1378eb2b)